src/main/java/com/xceptance/xlt/engine/httprequest/HttpRequest.java

```diff
@@ -362,22 +362,41 @@ public HttpRequest removeHeaders()
      */
     public HttpRequest param(final String name, String value)
     {
+        param(new NameValuePair(name, value));
+
+        return this;
```

Comment:
As this is just a delegate to the overloaded `param` method, I'd prefer to change its implementation to
`return param(new NameValuePair(name, value));`
WDYT?

Reply:
True.